helix-term/src/config.rs

```diff
@@ -12,7 +14,7 @@ pub struct Config {
     #[serde(default)]
     pub lsp: LspConfig,
     #[serde(default)]
-    pub keys: Keymaps,
+    pub keys: HashMap<Mode, Keymap>,
```

Comment:
The "Keymaps" struct is now divorced from the config. Since the `Keymaps` encapsulated view state alongside config, it wasn't quite fitting, and had to be changed for config projection to work accordingly.